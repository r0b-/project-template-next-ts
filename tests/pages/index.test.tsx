import {shallow} from "enzyme";
import * as React from "react";
import IndexPage from "../../pages/index";

describe("Index page", () => {
    it("should render correctly", () => {
        const wrapper = shallow(<IndexPage/>);

        expect(wrapper.html()).toMatchSnapshot();
    });
});
