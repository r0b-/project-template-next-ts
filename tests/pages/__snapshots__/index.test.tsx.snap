// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index page should render correctly 1`] = `"<nav class=\\"sc-bdVaJa hioVum\\"><a href=\\"/\\">Home</a><a href=\\"/about\\">About</a></nav><h1>Index</h1>Index Content<footer><div>Copyright 2017</div></footer>"`;
